### PR TITLE
starting to update starter kit to wepack 5

### PR DESCRIPTION
**Pull request by adricen**
migration from wepack 4 to webpack 5

Still have poliffil to deal with in another time - package is no more updated.

A good alternative seams to be well explain here : [the-end-of-babel-polyfill-the-more-efficient-alternative](https://karen-kua.medium.com/the-end-of-babel-polyfill-the-more-efficient-alternative-95f959fef9c2)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "src/index.js",
   "scripts": {
     "start": "node server",
-    "dev": "cross-env NODE_ENV=dev node server",
+    "dev": "concurrently --kill-others \"cross-env NODE_ENV=dev node server\"",
```
Comment (flozz) [inline package.json]:
```
$ npm run dev

> starter@0.1.0 dev
> concurrently --kill-others "cross-env NODE_ENV=dev node server"
```

There is probably a missing dev dependency :)

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
...
     "release": "cross-env NODE_ENV=production webpack --config ./config/webpack.config.release.js && cp -r server build/release && cp package.json build/release && cd build/release && npm install --only=prod --ignore-scripts",
```
Comment (flozz) [inline package.json]:
```
$ npm run release 

> starter@0.1.0 release
> cross-env NODE_ENV=production webpack --config ./config/webpack.config.release.js && cp -r server build/release && cp package.json build/release && cd build/release && npm install --only=prod --ignore-scripts

[webpack-cli] Failed to load '.../obsidianjs-starter/config/webpack.config.release.js' config
[webpack-cli] Invalid options object. Terser Plugin has been initialized using an options object that does not match the API schema.
 - options has an unknown property 'sourceMap'. These properties are valid:
   object { test?, include?, exclude?, terserOptions?, extractComments?, parallel?, minify? }
```

The release config has probably been removed? :)

**Review by flozz — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (adricen) [reply package.json]: I downgraded Terser webpack plugin to v4 that seams to solved sthe problem

File: package.json (hunk 1 above)
Comment (adricen) [reply package.json]: I removed it

**Pull request closed without merging**